pkg/query/storeset.go

```diff
@@ -349,6 +357,7 @@ func (s *StoreSet) updateStoreStatus(store *storeRef, err error) {
 	s.storesStatusesMtx.Lock()
 	defer s.storesStatusesMtx.Unlock()
 
+	s.storeNodeStatus.With(prometheus.Labels{"node": store.addr}).Set(0.0)
```

Comment:
Instead of this you can do:
```
s.storeNodeStatus.WithLabelValues(store.addr).Set(0)
```
it's a bit more concise :) what do you think?

Reply:
Done :)